Source/WebCore/page/Navigation.cpp

```diff
@@ -613,6 +613,18 @@ void Navigation::abortOngoingNavigation(NavigateEvent& event)
     }
 }
 
+static NavigationHistoryBehavior navigationTypeToHistoryBehavior(NavigationNavigationType navigationType)
+{
+    switch (navigationType) {
+    case NavigationNavigationType::Push:
+        return NavigationHistoryBehavior::Push;
+    case NavigationNavigationType::Replace:
+        return NavigationHistoryBehavior::Replace;
+    default:
+        return NavigationHistoryBehavior::Auto;
+    }
```

Comment:
Here, you may want to include ASSERT_ NOT_REACHED logic, you can check how similar helper functions do that.

Reply:
I feel it isn't necessary for one with a `default` personally.